astropy_helpers/commands/build_sphinx.py

```diff
@@ -35,25 +31,17 @@
 def ensure_sphinx_astropy_installed():
     """
     Make sure that sphinx-astropy is available.
-
-    This returns the available version of sphinx-astropy as well as any
-    paths that should be added to sys.path for sphinx-astropy to be available.
     """
-    # We've split out the Sphinx part of astropy-helpers into sphinx-astropy
-    # but we want it to be auto-installed seamlessly for anyone using
```

Comment:
probably worth mentioning in the changelog that there is no more auto installing

Reply:
This was actually done in https://github.com/astropy/astropy-helpers/pull/474 for 3.2 - I'm just removing leftover comments.